addons/bus/controllers/websocket.py

```diff
@@ -66,5 +66,7 @@ def get_websocket_worker_bundle(self, v=None):  # pylint: disable=unused-argumen
         bundle = 'bus.websocket_worker_assets'
         files, _ = request.env["ir.qweb"]._get_asset_content(bundle)
         asset = AssetsBundle(bundle, files)
-        stream = request.env['ir.binary']._get_stream_from(asset.js())
+        stream = request.env['ir.binary']._get_stream_from(asset.js(
+            is_minified=request.session.data.get("debug") != "assets"
```

Comment:
`data` is the underlying, implementation-specific, datastructure. You shouldn't use it outside of the Session classe
```suggestion
            is_minified="assets" in request.session.debug
```

Reply:
huh sorry I think it should be `not in` :see_no_evil:

Reply:
and don't forget to rebase your branch

Reply:
Ready.

Reply:
I'm waiting for the runbot :p